.github/workflows/release.yml

```diff
@@ -23,10 +23,14 @@ jobs:
       - uses: actions/setup-python@v4
         with:
           python-version: "3.11"
-      - name: Create Wheel
+      - name: Clean
         run: |
-          pip install wheel
-          pip wheel . --no-deps --wheel-dir dist
+          python -c 'import shutil; [shutil.rmtree(p, True) for p in ("build", "dist", "docs/_build")]'
```

Comment:
Why is the `docs/_build` folder here? Isn't everything done when building the docs either moved to the build folder or deleted?

Reply:
this is a blatant copy from our friends over at foundry dev tools, the docs fodler is prob not necessary, but it will not exist and not change everything here anyway

Reply:
Still, if we do not have such a folder, we should not have a command for deleting it here.

Reply:
thy wish is my command